dev/app/app.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
 /**
+ * Change to switch between local JSON and firebase
+ */
+var localData = true;
+
```

Comment:
I would go back regarding this. Keeping variable in separate file and have two task gulp for dev and dist.